internal/actions/broker.go

```diff
@@ -102,20 +115,43 @@ func (c *Container) BrokerDeploy(ctx *cli.Context) error {
 	}
 	if err := sshClient.CopyFilesOverSftp(
 		conn.SftpCopySrcDest{Src: chainConfig, Dst: "./broker/chainConfig.json"},
+		conn.SftpCopySrcDest{Src: rpcConfig, Dst: "./broker/rpc.json"},
 		conn.SftpCopySrcDest{Src: dockerCompose, Dst: "./broker/docker-compose.yml"},
+		conn.SftpCopySrcDest{Src: keyfile, Dst: "./broker/keyfile.txt"},
 	); err != nil {
+		os.Remove(keyfile)
 		return err
 	}
+	os.Remove(keyfile)
 
 	// Exec broker-server deployment cmd
-	fmt.Println(styles.ItalicText.Render("Starting docker compose on broker-server..."))
-	cmd := "cd ./broker && echo '%s' | sudo -S BROKER_KEY=%s BROKER_FEE_TBPS=%s REDIS_PW=%s docker compose up -d"
+	fmt.Println(styles.ItalicText.Render("Preparing Docker volumes..."))
+	cmd := "cd ./broker && echo '%s' | sudo -S docker volume create broker_mydata "
+	cmd = cmd + "&& echo '%s' | sudo -S docker run --rm -v $PWD:/source -v broker_mydata:/dest -w /source alpine cp ./keyfile.txt /dest"
```

Comment:
default user `d8xtrader` is now by default added to `docker` group, so `echo 'password'...` for docker commands is not needed anymore (of course only for newly provisioned servers with latest dev cli version). But we should now generally stop using `echo passwd` approach for docker commands.

Reply:
https://github.com/D8-X/d8x-cli/blob/dev/internal/configs/embedded/playbooks/setup.ansible.yaml#L123